client/components/FishDetails.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {Container, Grid, Icon, Image, Header, Divider, Button, Placeholder} from 'semantic-ui-react'
+import {Link} from 'react-router-dom'
+
+const FishDetails = ({match}) => {
+
+  const name = (match.params.name)
+
+  return (
+    <div>
+      <Container>
+        <Grid columns={3} doubling stackable>
+          <Grid.Column>
+            <Link to='/'><Icon color='green' name='arrow circle left' size='huge' /></Link>
```

Comment:
The Home button in the menu already performs this function (going back to the root). It might be better to remove the `<Link>` and just add an onclick to the `<Icon>`, e.g.:
```js
onClick={() => history.back()}
```
That way, if another sequence of events leads them to view the `<FishDetail>` component, the Back button will just send them back one click.

Reply:
Have now changed the function of the back arrow icon to the above suggestion.